android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/RibCoroutineWorker.kt

```diff
@@ -208,15 +208,16 @@ internal open class RibCoroutineWorkerToWorkerAdapter(
   private val ribCoroutineWorker: RibCoroutineWorker,
   private val dispatcher: CoroutineDispatcher?,
```

Comment:
Now that we're using `CoroutineContext` in `Worker`, we can receive a `CoroutineContext` in here instead, and do:
```
  override val coroutineContext: CoroutineContext = coroutineContext + super.coroutineContext
```

`super.coroutineContext` will override elements already present in `this.coroutineContext` (if any)

Reply:
good catch. Will update